ReSharper.FSharp/src/FSharp.Common/src/Util/FSharpHighlightingAttributeIds.fs

```diff
@@ -98,6 +103,22 @@ type FSharpSettingsNamesProvider() =
       Layer = HighlighterLayer.SYNTAX,
       EffectType = EffectType.TEXT, ForegroundColor = "#000000", DarkForegroundColor = "#B5CEA8");
 
+  RegisterHighlighter(
+      FSharpHighlightingAttributeIds.Measure,
+      FallbackAttributeId = FSharpHighlightingAttributeIds.TypeParameter,
+      GroupId = FSharpHighlightingAttributeIds.GroupId,
+      RiderPresentableName = "Syntax//Measure",
```

Comment:
It should probably go to the `Types` group and to have `Unit of measure` name that is used in F# docs.

Reply:
Fixed.